imos_bgc_db/NRS_Phyto_Raw.schema.yaml

```diff
@@ -0,0 +1,27 @@
+fields:
+  - name: SAMPLE
+    description: Sample id (references NRS_Samp table)
+    type: integer
+  - name: TAXON_NAME
+    type: string
+  - name: TAXON_GROUP
+    type: string
+  - name: GENUS
+    type: string
+  - name: SPECIES
+    type: string
+  - name: SPCODE
+    type: integer
+  - name: CELL_L
+    description: Number of cells per Litre
+    type: number
+  - name: BIOVOLUME_UM3L
+    description: Biovolume (cubic microns per Litre)
+    type: number
+primaryKey:
+  - SAMPLE
+  - TAXON_NAME
```

Comment:
These fields are not allowed to have null values, and each combination needs to be unique.